src/bci_build/package/rust.py

```diff
@@ -79,6 +85,9 @@
 # smoke test that gcc works
 RUN gcc --version
 RUN ${{CC}} --version
+COPY {check_fname} /etc/zypp/systemCheck.d/{check_fname}
+# check that {check_fname} contains a valid requires
+{DOCKERFILE_RUN} zypper -n verify; zypper -n clean; rm -rf /var/log/*
```

Comment:
It would do something sensible, if obs wouldn't run with network isolation. Now the build fails instead: https://build.opensuse.org/package/live_build_log/home:defolos:BCI:Staging:Tumbleweed:Tumbleweed-vFsvP/rust-oldstable-image/containerfile/x86_64.

I will drop the `zypper verify` hence